processing/data_collection/gazette/spiders/mt_cuiaba.py

```diff
@@ -0,0 +1,54 @@
+import json
+import time
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
```

Comment:
Add a space between standard library and third party libraries. See [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports)